feat: integrate react-native-brownfield (#271)
callstackincubator/rock: website/docs/docs/brownfield/ios.mdx

   - `$(SRCROOT)/.xcode.env.local`
   - `$(SRCROOT)/.xcode.env`

### 4. Create the Framework's Public Interface:

1. Install `react-native-brownfield`:
   <PackageManagerTabs command="install @callstack/react-native-brownfield" />

> 1. Create a new Swift file in your framework folder using Xcode[^1]
> 1. Copy and paste the contents of [HelloWorldReact.swift](https://github.com/callstack/rnef/tree/main/packages/plugin-brownfield-ios/template/ios/HelloWorldReact/HelloWorldReact.swift) to your Swift file

Reviewer: ````suggestion
1. Create a new Swift file in your framework folder using Xcode with the following contents:
   ```swift
   @_exported import ReactBrownfield
   ```
````
